### ISSUE TITLE
Update download Engine link to point to Pardot form

### ISSUE THREAD

**Issue by vcoisne** [src-d/landing#347]
Per https://github.com/src-d/feature-idea/issues/88 Every engine dowload links on https://sourced.tech/ should now point to https://go.sourced.tech/engine-download instead of https://docs.sourced.tech/engine/#quickstart

Here is the pages / links that need to be updated:

- On the homepage / every page in the header menu CTA button and bottom of every page CTA
- On https://sourced.tech/engine/ "install Engine" button in the hero banner and "get started" button at the bottom of the page

**Comment by ricardobaeta** [comment]
@dpordomingo @vcoisne I'll update the Logo on the Header and Footer, and the typography colours on the latter, on Pardot asap.

> This applies to all Pardot Landings.

**Comment by vcoisne** [comment]
thanks @ricardobaeta, forgot about the new logo :)

**Comment by dpordomingo** [comment]
Just giving my opinion as a user.

There are these two buttons on https://sourced.tech/engine/

![image](https://user-images.githubusercontent.com/2437584/51384931-0d712b00-1b1e-11e9-9282-28a66f33d072.png)

- `Request a demo` points to https://go.sourced.tech/engine
- `Install source{d} Engine` points to https://docs.sourced.tech/engine#quickstart

Should both of them poit to the very same place?
Because I'd find it quite disturbing.
Is it really helpful for a developer trying to use source{d} Engine?

Anyhow, I tried with the `Request a demo` form, sending my details and email and I was redirected to source{d} homepage; I waited a bit but I didn't receive an email explaining what should be next. If I try again, going to `go.sourced.tech/engine`, I'm automatically redirected to source{d} homepage what is weird.

**Comment by vcoisne** [comment]
Hey @dpordomingo, thanks for the feedback. 

They are both landing pages but not the same. 

1. Request a demo = http://go.sourced.tech/engine

2. Install = http://go.sourced.tech/engine-download

I updated the completion action of the "request a demo" landing to make sure there is a thank you message and allow users to return to the landing page after completion. Once they fill out the form either Hass, Jean or I follow up with them with more info and next steps. 

For the install landing page, people who fill out the form get an automated email with the link to the latest version of the Engine based on their respective OS. 

@ricardobaeta I updated the logo in the automated email users receive once filling out the install landing page. Would be awesome if you can update it for the actual landing pages.

**Comment by ricardobaeta** [comment]
Hi @vcoisne ! This [PR](https://github.com/src-d/landing/pull/348) that you can actually see on https://landing-staging.srcd.run/ (with a couple more changes) is about to be merged. As soon as it is I'll update the Pardot Landing Template, that will update all Landings.

**Comment by dpordomingo** [comment]
thanks, @vcoisne for the feedback!

yes, having two different links lgtm (I didn't realize about they were already different)

I tried https://go.sourced.tech/engine-download with two different emails, and I got the `Thank you!'' message, saying you will recieve an email with the download link...` But ~10 mins later I received nothing (it's weird: nor inbox nor spam).

I'd recommend adding a link to the github project page in the `Thank you` message, so the user can click there to see the sources in case something goes wrong with the email.

Also, how're you managing the release version that is emailed? using `:latest` tag?

**Comment by vcoisne** [comment]
@dpordomingo I just tried and got the email 1m later. Try again with a different email in incognito mode ? 

Not using the latest tag, I will have to update the release link manually I guess.

**Comment by dpordomingo** [comment]
I tried with two different emails with no success, and then I tried with a third one successfully.
Then, I tried again with the third one with no success.

Might be the reason that two first attempts were with an already used email, and the third one with a brand new email?
If so, is this the expected behavior?
What if a visitor (that already shared their email) wants to download one of our products?

(sending emails via slack)
